### PR TITLE
[Merged by Bors] - Book modifications to include testnet param

### PR DESCRIPTION
**Pull request by AgeManning**
## Issue Addressed

N/A

## Proposed Changes

Update the lighthouse book to accommodate the new `--testnet` CLI param.

### CODE REVIEW

File: book/src/validator-create.md [hunk 1]
```diff
@@ -60,7 +66,7 @@ The example assumes that the `wally` wallet was generated from the
 [wallet](./wallet-create.md) example.
 
 ```bash
-lighthouse account wallet validator --name wally --wallet-password wally.pass
+lighthouse --testnet medalla account wallet validator --name wally --wallet-password wally.pass
```
Comment (pawanjay176) [inline book/src/validator-create.md:69]: this needs to be 
`
lighthouse --testnet medalla account  validator create --wallet-name wally --wallet-passphrase wally.pass --count 1
`

Comment (AgeManning) [reply book/src/validator-create.md:69]: Does it not default to `medalla` ?

I left some of these to the default to avoid over complicating the commands.

Comment (michaelsproul) [reply book/src/validator-create.md:69]: Pawan's pointing out the `--wallet-password` vs `--wallet-passphrase` flag

Comment (pawanjay176) [reply book/src/validator-create.md:69]: Yep, we have had the wrong command in the book for a while. :sweat_smile: 
It should be `account validator create` and not `account wallet validator`, `wallet-passphrase` and not `wallet-password`, `wallet-name` and not `name` and `--count/--at-most` is required param.
It's correct in the help command output above though.

Comment (michaelsproul) [reply book/src/validator-create.md:69]:
```suggestion
lighthouse --testnet medalla account validator create --name wally --wallet-passphrase wally.pass --count 1
```

**Pull request closed without merging**